From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Fri, 7 Feb 2025 18:51:25 +0100
Subject: test: log error when constructing sysroot

Log dracut errors for all dracut invocations. Otherwise tests can fail
without any log output.

Fixes: 8631284afbfa ("ci: do not log how sysroot is constructed")
Forwarded: https://github.com/dracut-ng/dracut-ng/pull/1201
---
 test/dracut.conf.d/test-makeroot/test-makeroot.conf | 2 +-
 test/dracut.conf.d/test-root/test-root.conf         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/dracut.conf.d/test-makeroot/test-makeroot.conf b/test/dracut.conf.d/test-makeroot/test-makeroot.conf
index 48373e0..d6a0067 100644
--- a/test/dracut.conf.d/test-makeroot/test-makeroot.conf
+++ b/test/dracut.conf.d/test-makeroot/test-makeroot.conf
@@ -5,4 +5,4 @@ do_strip="no"
 do_hardlink="no"
 early_microcode="no"
 hostonly_cmdline="no"
-stdloglvl=1
+stdloglvl=2
diff --git a/test/dracut.conf.d/test-root/test-root.conf b/test/dracut.conf.d/test-root/test-root.conf
index 9ebf4a59..2bb46dc 100644
--- a/test/dracut.conf.d/test-root/test-root.conf
+++ b/test/dracut.conf.d/test-root/test-root.conf
@@ -5,4 +5,4 @@ do_strip="no"
 do_hardlink="no"
 early_microcode="no"
 hostonly_cmdline="no"
-stdloglvl=1
+stdloglvl=2
